wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/en.md

```diff
@@ -0,0 +1,77 @@
+# Handling Foul Play
+
```

Comment:
Can you put: `_Main page: [Reporting Bad Behaviour](/wiki/Reporting_Bad_Behaviour)._` here?

Reply:
Done.